src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,16 @@
+package Duke;

```

Comment:
Good job packaging all your classes into a package! Remember that package names should be in lower case